core/templates/dev/head/components/share/sharing_links_directive.html

```diff
@@ -24,7 +24,7 @@
   </li>
 
   <li ng-if="shareType === 'exploration'">
-    <a ng-click="showEmbedExplorationModal(explorationId)" uib-tooltip="Embed" tooltip-placement="top" tabindex="0">
+    <a href="" ng-click="showEmbedExplorationModal(explorationId)" uib-tooltip="Embed" tooltip-placement="top" tabindex="0">
```

Comment:
If you just do `href` (without the `=""`), will it work? I think we do that elsewhere.

Reply:
Yes, But in order to make it work in other browsers like Safari and IE there must be a value for href and it can be an empty string.

Reply:
Oh, interesting. Does that mean that when we use just `href` elsewhere, it's wrong and should be fixed?

Reply:
(grep for `a href `, with the space at the end)

Reply:
yes, there are 7 occurrences of href without value in search_bar_directive and search_results_directive.

Reply:
Ah ok. Would you mind adding this there too, then, so that the whole codebase is consistent?